users/forms.py

```diff
@@ -2,7 +2,7 @@
 from django.contrib.auth.models import User
 from django.contrib.auth.forms import UserCreationForm
 from ideas.manager import email_verification
-
+from snowpenguin.django.recaptcha3.fields import ReCaptchaField
```

Comment:
For ordering imports, try to follow this rule
- python packages
- django packages
- third-party packages
- user-defined packages